File: java/arrays/Richest_Customer_Wealth_1672.java
package java.arrays;

public class Richest_Customer_Wealth_1672 {
        public int maximumWealth(int[][] accounts) {
        int wealth=0;
        for(int i=0;i<accounts.length;i++){
            int sum=0;
            for(int j=0;j<accounts[i].length;j++){
                sum+=accounts[i][j];
            }
            if(sum>wealth){
                wealth=sum;
            }
        }
        return wealth;
    }
}

File: python/arrays/Richest_Customer_Wealth_1672.py
class Solution:
    def maximumWealth(self, accounts: List[List[int]]) -> int:
        wealth=0
        for i in range(len(accounts)):
            sum=0
            for j in range(len(accounts[i])):
                sum+=accounts[i][j]
            if sum>wealth:
                wealth=sum
        return wealth
            